css/src/components/calendar.css

```diff
@@ -177,19 +189,26 @@
   background: var(--alert-lightest);
 }
 
-.Calendar-value--currDateMonthYear,
+.Calendar-value--currDateMonthYear {
+  background: var(--primary-lightest);
+}
+
 .Calendar-value--currDateMonthYear:hover {
-  background: var(--primary-lightest) !important;
+  background: var(--primary-lighter);
 }
 
-.Calendar-value--active,
-.Calendar-value--active:hover {
-  background: var(--primary) !important;
+.Calendar-value--currDateMonthYear:active {
+  background: var(--primary-light);
+}
+
+.Calendar-value--active {
+  background: var(--primary);
   font-weight: var(--font-weight-bold);
 }
 
-.Calendar-yearValue,
-.Calendar-monthValue {
+.Calendar-value--active:hover {
+  background: var(--primary-dark) !important;
+  font-weight: var(--font-weight-bold);
```

Comment:
There is no need to define font-weight again. It will pick from the parent class i.e. **Calendar-value--active**.